docs/assets/stylesheets/extra.css

```
@@ -0,0 +1,65 @@
+@keyframes headerGlow {
```

Stylelint (17.13.0)

[error] Expected keyframe name "headerGlow" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)